### PR TITLE
Cuda 10.2 cross compile for nano 

### PR DESCRIPTION
**Pull request by agibsonccc**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

## Quick checklist

The following checklist helps ensure your PR is complete:

- [ ] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [ ] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.

### CODE REVIEW

File: .github/workflows/build-deploy-linux-cuda-10.2-arm64.yml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+      deployToReleaseStaging:
+        description: 'Whether to deploy to release staging or not.'
+        required: false
+        default: 0
+
+      releaseVersion:
+        description: 'Release version target'
+        required: false
+        default: 1.0.0-M1
```
Comment (treo) [inline .github/workflows/build-deploy-linux-cuda-10.2-arm64.yml:17]: Shouldn't this be 1.0.0-M1.1?

**Review by treo — CHANGES_REQUESTED** [review]

File: libnd4j/pi_build.sh [hunk 2]
```diff
@@ -222,12 +225,19 @@ function fix_pi_linker {
 function cuda_cross_setup {
 		# $1 is local  cuda toolkit version
 		# $2 the folder where cross cuda toolkit will be
-		loc_VER="${1-10.2}"
+		loc_VER="${LOCAL_CUDA_INSTALLED_VER}"
 		loc_DIR="${2-/tmp}"
 		CUDA_DIR="${CUDA_DIR-/usr/local/cuda-${loc_VER}}"
 		CUDA_TARGET_STUBS="${loc_DIR}/target_stub${loc_VER}"
-		CUDA_TARGET_STUB_URL=https://github.com/quickwritereader/temp_jars/releases/download/1.2/aarch64_linux_cuda_${loc_VER}_cudnn8.tar.gz
+		CUDA_TARGET_STUB_URL=https://github.com/KonduitAI/jetson-release/releases/download/0.1.0/aarch64_linux_cuda_10.2_cudnn8.2.tar.gz
```
Comment (quickwritereader) [inline libnd4j/pi_build.sh:232]: it would be great if you had a generated URL based on versions here.

File: libnd4j/pi_build.sh [hunk 3]
```diff
@@ -222,12 +225,19 @@ function fix_pi_linker {
...
 		download_extract "${CUDA_TARGET_STUB_URL}" "${CUDA_TARGET_STUBS}"
+		mv "${CUDA_TARGET_STUBS}/aarch64_linux_cuda_10.2_cudnn8.2" "${CUDA_TARGET_STUBS}/aarch64-linux/"
+		message "Moving  ${CUDA_TARGET_STUBS}/aarch64_linux_cuda_10.2_cudnn8.2 to ${CUDA_TARGET_STUBS}/aarch64-linux/"
```
Comment (quickwritereader) [inline libnd4j/pi_build.sh:235]: the same here, hardcoded versions

**Review by quickwritereader — APPROVED** [review]
though I believe it would be great if you changed hardcoded URL and versions to generated/external arguments 
but looks good to me

File: .github/workflows/build-deploy-linux-cuda-10.2-arm64.yml (hunk 1 above)
Comment (agibsonccc) [reply .github/workflows/build-deploy-linux-cuda-10.2-arm64.yml:17]: I can do a separate PR for the next version. For now I just need to test this ci work flow. It's a new file.

**Review by treo — APPROVED** [review]

**Pull request merged**